src/ResourceManager/Websites/Commands.Websites/Models.WebApp/PSSite.cs

```diff
@@ -55,7 +55,6 @@ public PSSite(Site other)
                   possibleOutboundIpAddresses: other.PossibleOutboundIpAddresses,
                   dailyMemoryTimeQuota: other.DailyMemoryTimeQuota,
                   suspendedTill:other.SuspendedTill,
-                  snapshotInfo: other.SnapshotInfo,
```

Comment:
Why is this being removed? Should the property be deprecated?

Reply:
This property was defined in our APIs but never used for anything. It was removed in the newest API version. There's almost no chance that a customer had a dependency on this property, but we can deprecate it in whatever way is needed.

Reply:
@maddieclayton wanted to make sure you agree with our plan to change the PSSite.cs wrapper to not extend the Site SDK model, we believe this will also fix our breaking changes comment you had raised as well. Let me know if you have other suggestions here.

Reply:
Sounds good! You could also extend the SDK model but simply add back in the removed properties.  Up to you.